modules/images-other-jenkins-config-kubernetes.adoc

```diff
@@ -159,20 +154,20 @@ data:
 
 [NOTE]
 ====
-If you log in to the Jenkins console and make further changes to the pod template configuration after the pod template is created, and the {product-title} Sync plug-in detects that the config map has changed, it will replace the pod template and overwrite those configuration changes. You cannot merge a new configuration with the existing configuration.
+If you log in to the Jenkins console and make further changes to the pod template configuration after the pod template is created, and the {product-title} Sync plugin detects that the config map has changed, it will replace the pod template and overwrite those configuration changes. You cannot merge a new configuration with the existing configuration.
 
-Do not log in to the Jenkins console and change the pod template configuration. If you do so after the pod template is created, and the {product-title} Sync plug-in detects that the image associated with the image stream or image stream tag has changed, it replaces the pod template and overwrites those configuration changes. You cannot merge a new configuration with the existing configuration.
+Do not log in to the Jenkins console and change the pod template configuration. If you do so after the pod template is created, and the {product-title} Sync plugin detects that the image associated with the image stream or image stream tag has changed, it replaces the pod template and overwrites those configuration changes. You cannot merge a new configuration with the existing configuration.
 
 Consider the config map approach if you have more complex configuration needs.
 ====
 
-After it is installed, the {product-title} Sync plug-in monitors the API server of {product-title} for updates to image streams, image stream tags, and config maps and adjusts the configuration of the Kubernetes plug-in.
+After it is installed, the {product-title} Sync plugin monitors the API server of {product-title} for updates to image streams, image stream tags, and config maps and adjusts the configuration of the Kubernetes plugin.
 
 The following rules apply:
 
-* Removing the label or annotation from the config map, image stream, or image stream tag results in the deletion of any existing `PodTemplate` from the configuration of the Kubernetes plug-in.
-* If those objects are removed, the corresponding configuration is removed from the Kubernetes plug-in.
+* Removing the label or annotation from the config map, image stream, or image stream tag results in the deletion of any existing `PodTemplate` from the configuration of the Kubernetes plugin.
+* If those objects are removed, the corresponding configuration is removed from the Kubernetes plugin.
 * Either creating appropriately labeled or annotated `ConfigMap`, `ImageStream`, or `ImageStreamTag` objects, or the adding of labels after their initial creation, leads to creating of a `PodTemplate` in the Kubernetes-plugin configuration.
```

Comment:
I know this isn't in the review, but it looks like a candidate for the passive voice - it would improve the readability significantly. "creating" and "the adding" feel a bit forced here - I would definitely drop "the" from "the adding" at least.

Or simplify to "If you create...or add..", this results in the creation of...."

Reply:
Thanks. Updated.